test/fixtures/basic-worker/index.js

```diff
@@ -0,0 +1,20 @@
+const pipe = Pear.worker.pipe()
+
+pipe.on('data', async (data) => {
+  const type = data.toString()
```

Comment:
we can simplify this, we dont need to include the type in the response because test requests are going to be awaited, so there is no need to check the type from the test

Reply:
updated

Reply:
updated